src/ripple/test/WSClient.h

```diff
@@ -47,7 +47,7 @@ class WSClient : public AbstractClient
 
 /** Returns a client operating through WebSockets/S. */
 std::unique_ptr<WSClient>
-makeWSClient(Config const& cfg, bool v2 = true);
+makeWSClient(Config const& cfg, bool v2 = true, unsigned rpc_version = 2);
```

Comment:
Would `@param`s docs be worthwhile here? It's pretty obvious by the variable names, but I don't know how thorough we want to be.